streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SettingServiceImpl.java

```diff
@@ -81,6 +74,9 @@ public boolean update(Setting setting) {
             if (setting.getSettingKey().equals(CommonConfig.MAVEN_AUTH_PASSWORD().key())) {
                 InternalConfigHolder.set(CommonConfig.MAVEN_AUTH_PASSWORD(), value);
             }
+            if (settings == null || settings.size() == 0) {
+                initSetting();
+            }
             settings.get(setting.getSettingKey()).setSettingValue(value);
```

Comment:
if not, suggest use optional to resolve npe. btw, settings is not null, refer to 51 line.
```suggestion
            Optional<Setting> optional = Optional.ofNullable(settings.get(setting.getSettingKey()));
            if (optional.isPresent()) {
                optional.get().setSettingValue(value);
            }
```

Reply:
@lvshaokang Thanks for your review, addressed.